app/models/category_transaction.rb

```diff
@@ -0,0 +1,4 @@
+class CategoryTransaction < ApplicationRecord
+  belongs_to :category
+  has_many :transaction
```

Comment:
Required

-  Following the association issue above, in this third model that connects the two main models to each other should only belong to them. So, please modify the has many association to belongs to